fvm/src/call_manager/mod.rs

```diff
@@ -111,6 +119,15 @@ pub trait CallManager: 'static {
         delegated_address: Option<Address>,
     ) -> Result<()>;
 
+    // returns the actor call stack
+    fn get_actor_call_stack(&self) -> &Vec<(ActorID, &'static str)>;
+
+    /// add an actor to the calling stack.
+    fn actor_call_stack_push(&mut self, actor_id: ActorID, entrypoint: &Entrypoint);
+
+    /// pop an actor from the calling stack.
+    fn actor_call_stack_pop(&mut self) -> Option<(ActorID, &'static str)>;
```

Comment:
Can we avoid adding these to the interface?

Reply:
definetly!